doc/tutorials/elektra-cmerge-integration.md

```diff
@@ -0,0 +1,14 @@
+# How-To: Integrate elektra-merge Into a Debian Package
```

Comment:
As discussed, better to leave this away. But a tutorial for the standalone merging tool would be interesting.

Reply:
Removed the package part. I'll add the standalone tutorial in a new tutorial so that this PR is ready to merge asap.